index.bs

```diff
@@ -0,0 +1,331 @@
+          <td>
+            "<dfn>allowed-muted</dfn>"
+          <td>
+            Inaudible media element</dfn> are allowed to autoplay. An
+            <dfn export>inaudible media element</dfn> is an {{HTMLMediaElement}}
+            that has any of those conditions true:
+            <ol>
+            <li> media's {{HTMLMediaElement/volume}} equal to 0
+            <li> media's {{HTMLMediaElement/muted}} is true
+            <li> media's <a href="https://html.spec.whatwg.org/multipage/media.html#media-resource">resource</a> does not have an audio track
+            </ol>
+        <tr>
+          <td>
+            "<dfn>disallowed</dfn>"
+          <td>
+            No media is allowed to autoplay.
+    </table>
+    </div>
+
+    <div class="note">
+      Note: The autoplay policy represents the current status of whether a
+      user-agent allows media to autoplay, which can **vary** in the future.
+      Therefore, it is **recommended** authors to check the result every time
+      they want to have an up-to-date result.
+    </div>
+
+    <div class=example>
+      If a user-agent uses the user activation, described in
+      [[HTML#user-activation-data-model]], to determine if the autoplay media
+      are allowed or not, and it blocks all autoplay by default
+      ({{AutoplayPolicy/disallowed}}). Then the policy can change to
+      {{AutoplayPolicy/allowed}} or {{AutoplayPolicy/allowed-muted}} after a
+      user performs a supported user gesture on the page or the media.
+    </div>
+
+  <h3 id="document-api">The Document API</h3>
+    <pre class="idl">
+    partial interface Document {
+      readonly attribute AutoplayPolicy autoplayPolicy;
+    };
+    </pre>
+    This represents a rough status of whether media that belong to this
+    {{Document}} are allowed to autoplay or not.
+
+    <dl class="switch">
+      <dt>If the value is {{allowed}}</dt>
+      <dd>
+        All media that belong to this document are allowed to autoplay.
+      </dd>
+
+      <dt>If the value is {{allowed-muted}}</dt>
+      <dd>
+        All {{inaudible media element}} that belong to this document are allowed
+        to autoplay.
+      </dd>
+
+      <dt>If the value is {{disallowed}}</dt>
+      <dd>
+        **None** of media belongs to this document are allowed to autoplay.
+      </dd>
+    </dl>
+
+    <div class="note">
+      Note: However, depending on the implementation, it's still possible for
+      some media that exist on the same document to be allowed to autoplay when
+      the document {{Document/autoplayPolicy}} returns {{disallowed}}. It is
+      **recommended** for authors to check each media element’s autoplayPolicy
+      in order to get a complete result when document’s autoplayPolicy is
+      disallowed, if authors want to use that as a result to determine whether
+      media can be started playing.
+    </div>
+
+    <div class=example>
+      If a user-agent does not allow any media to autoplay by default, but
```

Comment:
I'd like to suggest a slight rephrasing: Some user-agents may not allow any media to autoplay by default, but allow autoplay on those media elements which have been clicked by users. In this case, at first, both the document's [etc]

Reply:
done, will update it later.